oprf/oprf.go

```diff
@@ -91,43 +91,28 @@ func generateContext(id SuiteID) []byte {
 }
 
 // Serialize serializes a KeyPair elements into byte arrays.
-func (kp *KeyPair) Serialize() ([]byte, []byte) {
-	pubK := kp.pubK.Serialize()
-	privK := kp.PrivK.Serialize()
-
-	return pubK, privK
+func (kp *KeyPair) Serialize() []byte {
+	return kp.PrivateKey.Serialize()
```

Comment:
Why does it only serialise the Private Key now?

Reply:
Because the public key is (already) derived from the private key, so serializing it was unnecessary and probably harmful, especially since the previous deserialization code didn't check to see if the derived public key matched that which was provided as input to `Deserialize`.